client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -172,4 +175,15 @@ public abstract RssInputStream readPartition(
   public abstract boolean unregisterShuffle(String applicationId, int shuffleId, boolean isDriver);
 
   public abstract void shutDown();
+
+  public abstract int pushDataByteBuf(
```

Comment:
please add some description to the method.

Reply:
Tks. I have added descriptions